addons/payment/models/payment_acquirer.py

```diff
@@ -125,6 +125,29 @@ def _valid_field_parameter(self, field, name):
     show_done_msg = fields.Boolean(compute='_compute_view_configuration_fields')
     show_cancel_msg = fields.Boolean(compute='_compute_view_configuration_fields')
 
+    @api.model
+    def _neutralize_fields(self, provider, fields):
+        """ Helper to replace neutralize a acquirer accounts for a specific provider
+            :param str provider: name of provider
+            :param list fields: list of fiels to update to `dummy`
+        """
+        acquirers = self.search([('provider', '=', provider)])
+        neutralized = {field_name: 'dummy' for field_name in fields}
```

Comment:
```suggestion
        neutralized = {field_name: False for field_name in fields}
```
Use an empty value instead, since the acquirers have been disabled (and the required_if_provider only applied to enabled/test acquirers).
Furthermore, it ensures that when re-enabling acquirers on a neutralized db, valid credentials have been provided.